.travis.yml

```diff
@@ -31,6 +31,7 @@ before_script:
   - phpenv config-rm xdebug.ini
   - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
   # Install packages using composer
+  - composer self-update --1
```

Comment:
Note to self: We need separate PR to get CI working with composer 2

Reply:
I think it should be working if this is tagged https://github.com/ezsystems/ezpublish-kernel/commit/2e438043b7b1e644b8467bd9d84f9a2347625867

Reply:
Ok, that should afaik happen soon

Reply:
Now it works with composer 2